unity-client/Assets/Scripts/MainScripts/Debugging/Panels/StatsPanelToggler.cs

```diff
@@ -18,28 +17,19 @@ public class StatsPanelToggler : MonoBehaviour
         void Start()
         {
             text = GetComponent<Text>();
-            allProfilingEnabled = false;
-            deepProfilingEnabled = false;
-
-            if (allProfilingEnabled)
-                EnableProfiling();
-            else
-                DisableProfiling();
+            deepProfilingEnabled = true;
 
+            EnableProfiling();
             RefreshShortcutText();
         }
 
         void Update()
         {
-            UpdateAllProfilingInput();
             UpdateDeepProfilingInput();
 
-            if (allProfilingEnabled)
+            if (Input.GetKeyUp(KeyCode.P))
```

Comment:
Maybe we should manage this input action from the `InputController`, in order to have all the possible inputs centralized.

Reply:
Going to skip this for now because I need to merge this now, and this is debugging code, so is not that important to have the key handling through the `InputController`. Maybe we can revisit this improvement later.